recipes/heffte/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "heFFTe" %}
+{% set version = "2.4.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/icl-utk-edu/heffte/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 02310fb4f9688df02f7181667e61c3adb7e38baf79611d80919d47452ff7881d
+  patches:
+    # macOS build error in aligned_alloc
+    # https://github.com/icl-utk-edu/heffte/pull/49
+    - 49.patch
+
+build:
+  number: {{ build }}
+
+  # heFFTe depends on MPI, but we do not yet build a Windows-MPI that we trust
+  skip: true  # [win]
+
+  # add build string so packages can depend on specific mpi variants
+  # dependencies:
+  # `pkg * mpi_mpich_*` for MPICH
+  # `pkg * mpi_openmpi_*` for OpenMPI
+  # `pkg * mpi_*` for any mpi
+  {% set mpi_prefix = "mpi_" + mpi %}
+  string: {{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make  # [unix]
+    - pkgconfig
+  host:
+    - fftw * mpi_{{ mpi }}_*
+    - {{ mpi }}
+  run:
+    - fftw * mpi_{{ mpi }}_*
+    - {{ mpi }}
+
+test:
+  requires:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make  # [unix]
+    - pkgconfig
```

Might help with the failing test on Linux?
```suggestion
    - pkgconfig
   -  pthread-stubs  # [linux]
```